package.json

```diff
@@ -131,10 +131,11 @@
     "lint-css": "stylelint 'dist/mapbox-gl.css'",
     "open-changed-examples": "git diff --name-only mb-pages HEAD -- docs/_posts/examples/*.html | awk '{print \"http://127.0.0.1:4000/mapbox-gl-js/example/\" substr($0,33,length($0)-37)}' | xargs open",
     "test": "run-s lint lint-css test-unit test-flow",
-    "test-suite": "run-s test-render test-query",
+    "test-suite": "run-s test-render test-render-recycle-map test-query",
     "test-suite-clean": "find test/integration/*-tests -mindepth 2 -type d  -not \\( -exec test -e \"{}/style.json\" \\; \\) -print | xargs -t rm -r",
     "test-unit": "tap --reporter dot --no-coverage test/unit",
     "test-render": "node --max-old-space-size=2048 test/render.test.js",
+    "test-render-recycle-map": "node --max-old-space-size=2048 test/render.test.js --recycle-map",
```

Comment:
I think `yarn test-render --recycle-map` should work, which would make this unnecessary.